bevy_rapier2d 0.1.0 fails to build on WASM

I have a bevy game that runs both natively and in the browser using WASM. However, after adding bevy_rapier2d to my Cargo.toml, like this...

```
bevy_rapier2d = { version="0.10.0", features = ["wasm-bindgen"] } 
```

...the project fails to build for WASM (`cargo build --target wasm32-unknown-unknown --release`) with the following error:
```
   Compiling bevy_rapier2d v0.10.0
error[E0277]: `T` cannot be shared between threads safely
   --> ...\Cargo\registry\src\github.com-1ecc6299db9ec823\bevy_rapier2d-0.10.0\src\physics\resources.rs:309:19
    |
284 | pub trait PhysicsHooksWithQuery<UserData: WorldQuery>: Send + Sync {
    |                                                               ---- required by this bound in `resources::PhysicsHooksWithQuery`
...
309 | impl<T, UserData> PhysicsHooksWithQuery<UserData> for T
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `T` cannot be shared between threads safely
    |
help: consider further restricting this bound
    |
314 |     > + Sync,
    |       ^^^^^^

error[E0277]: `T` cannot be sent between threads safely
   -->..\Cargo\registry\src\github.com-1ecc6299db9ec823\bevy_rapier2d-0.10.0\src\physics\resources.rs:309:19
    |
284 | pub trait PhysicsHooksWithQuery<UserData: WorldQuery>: Send + Sync {
    |                                                        ---- required by this bound in `resources::PhysicsHooksWithQuery`
...
309 | impl<T, UserData> PhysicsHooksWithQuery<UserData> for T
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `T` cannot be sent between threads safely
    |
help: consider further restricting this bound
    |
314 |     > + Send,
    |       ^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0277`.
error: could not compile `bevy_rapier2d`

```
However, when I build it natively (`cargo build --release`), it works perfectly fine. Any ideas?

I tried both nightly and stable toolchains.

(This is on Windows btw, if that matters)

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -29,3 +29,14 @@ jobs:
       run: cargo test --verbose -p bevy_rapier2d;
     - name: Check bevy_rapier3d examples
       run: cargo test --verbose -p bevy_rapier3d;
+  build-wasm:
+    runs-on: ubuntu-latest
+    env:
+      RUSTFLAGS: -D warnings
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup target add wasm32-unknown-unknown
+      - name: build bevy_rapier2d
+        run: cd bevy_rapier2d && cargo build --verbose --features wasm-bindgen --target wasm32-unknown-unknown;
+      - name: build bevy_rapier3d
+        run: cd bevy_rapier3d && cargo build --verbose --features wasm-bindgen --target wasm32-unknown-unknown;

--- a/src/physics/resources.rs
+++ b/src/physics/resources.rs
@@ -309,9 +309,10 @@ pub trait PhysicsHooksWithQuery<UserData: WorldQuery>: Send + Sync {
 impl<T, UserData> PhysicsHooksWithQuery<UserData> for T
 where
     T: for<'a, 'b, 'c, 'd, 'e, 'f> PhysicsHooks<
-        RigidBodyComponentsSet<'a, 'b, 'c>,
-        ColliderComponentsSet<'d, 'e, 'f>,
-    >,
+            RigidBodyComponentsSet<'a, 'b, 'c>,
+            ColliderComponentsSet<'d, 'e, 'f>,
+        > + Send
+        + Sync,
     UserData: WorldQuery,
 {
     fn filter_intersection_pair(

